docs/postgres.md

```diff
@@ -6,11 +6,8 @@ describes how it can be configured.
 
 ## Building LND with postgres support
 
-To build LND with postgres support, include the following build tag:
-
-```shell
-⛰  make tags="kvdb_postgres"
-```
+To build LND with postgres support and for your specific platform, edit https://github.com/lightningnetwork/lnd/blob/1abe1da5b3a0e914b8c0f0d837fc0c17bed74a88/make/release_flags.mk#L41 by adding the `kvdb_postgres` to the end of the line.
```

Comment:
You can omit these instructions since it's postgres is now enabled by default for release builds: https://github.com/lightningnetwork/lnd/commit/68309d30c26b2a5f1a367177f7a278978334ab8a